test/integration/servicecidr/allocator_test.go

```diff
@@ -74,10 +74,10 @@ func TestServiceAllocation(t *testing.T) {
 			s1 := kubeapiservertesting.StartTestServerOrDie(t,
 				apiServerOptions,
 				[]string{
-					"--runtime-config=networking.k8s.io/v1beta1=true",
```

Comment:
did the test not need the v1beta1 API? if so, I'm surprised this was removed ... just emulating 1.33 wouldn't turn this on

Reply:
same question below where the `--runtime-config=networking.k8s.io/v1beta1=true` lines were removed

Reply:
Ah, so we didn't need to enable v1beta1 in all of these before. Ok.